agent/engine/docker_container_engine.go

```diff
@@ -726,7 +726,7 @@ func (dg *dockerGoClient) Stats(id string, ctx context.Context) (<-chan *docker.
 	go func() {
 		statsErr := client.Stats(options)
 		if statsErr != nil {
-			seelog.Warnf("Error retrieving stats for container %s: %v", id, statsErr)
+			seelog.Debugf("Error retrieving stats for container %s: %v", id, statsErr)
```

Comment:
We're losing errors from the `client.Stats` call if we log them at debug.  `container.processStatsStream` doesn't log anything at WARN (and doesn't really have much in the way of important errors) and `container.collect` just logs at DEBUG.